### DISCUSSION TITLE
Open new window in remaining space on screen

### DISCUSSION THREAD

**Discussion by robbins** [YaLTeR/niri#483] [Ideas]
Sometimes I'll shrink a window with Alt-, leaving a certain amount of space left on the current view of the screen (without scrolling left or right), and it'd be useful to have a keybind that I could toggle to open the next window in that remaining space.

![2024-06-29_06-42-46_PM](https://github.com/YaLTeR/niri/assets/31457698/67510013-ccd7-4af8-97b0-baf0e5dcc591)
e.g. on the far right in this screenshot

Would it be possible to implement this?

**Comment by YaLTeR** [comment]
So, like, with the default window width equal to that remaining size?

**Reply by robbins** [reply]
Yeah, exactly. But toggleable, so that windows open with the normal default-column-width by default, but default-column-width can be modified to be the remaining size (for the next window open) with a keybind.
